### PR TITLE
Remove AMQP code and Proton library dependencies & flags

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: contrib/debian/copyright [hunk 1]
```diff
@@ -1398,51 +1394,3 @@ License: Apache-Qpid-Proton-with-BSD-Subcomponents
    See the License for the specific language governing permissions and
    limitations under the License.
  .
```
Comment (daira) [inline contrib/debian/copyright:1396]: You can delete the whole of `License: Apache-Qpid-Proton-with-BSD-Subcomponents` (lines 1194..1413).

File: contrib/debian/copyright [hunk 2]
```diff
@@ -1398,51 +1394,3 @@ License: Apache-Qpid-Proton-with-BSD-Subcomponents
...
- of the source code for freegetopt is subject to the terms and
- conditions of its license in examples/include/pncompat/internal/LICENSE.
- .
- The setup scripts for the python bindings include files derived by
- PyZMQ and are licensed with a separate Modified BSD license.  Use of
- the source code in these setup files are subject to the terms and
- conditions in the license:
- proton-c/bindings/python/setuputils/PYZMQ_LICENSE.BSD.
-
-License: GNU-All-permissive-License
```
Comment (daira) [inline contrib/debian/copyright:1414]: This and `License: GPLv3-or-later-with-Autoconf-exception` are different licenses that should be retained.

Comment (daira) [reply contrib/debian/copyright:1414]: This hasn't been resolved.

**Comment by str4d** [comment]
@zkbot r+

**Comment by str4d** [comment]
Transient failure.

@zkbot retry

**Pull request merged**